CodeContracts.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.31101.0
+# Visual Studio 14
```

Comment:
:exclamation: We are not yet ready to make this change. The last time I checked, CodeContracts.sln has build failures if you try to compile it inside Visual Studio 2015. It should open by default in 2013 until we make a separate decision to move to 2015 for primary development.